style-guides/ruby/rails/controllers/specs/README.md

```diff
@@ -0,0 +1,3 @@
+## Specs
+
+* We need to think if use only controllers specs, request specs or acceptance tests.
```

Comment:
Unit test for all. And acceptance for interaction controllers and request spec for api controllers.

What you think?